relation.md

```diff
@@ -11,53 +11,51 @@ This is intended to be read in conjunction with the [Plonkish backend optimizati
 Plonkish arithmetisation depends on a scalar field over a prime modulus $p$. We represent this field as the object $\mathbb{F}$. We denote the additive identity by $0$ and the multiplicative identity by $1$. Integers, taken modulo the field modulus $p$, are called scalars; arithmetic operations on scalars are implicitly performed modulo p. We denote the sum, difference, and product of two scalars using the +, -, and * operators, respectively. 
 
 ## The Plonkish Relation
-The relation $\mathcal{R}_{\mathsf{plonkish}}$ contains pairs of public instances $\mathsf{instance}$ and private advice $w$.  We say that $\mathsf{instance}$ is a valid instance whenever their exists some advice $w$ such that $(\mathsf{instance}, w) \in \mathcal{R}_{\mathsf{plonkish}}$.  The Plonkish language $\mathcal{L}_{\mathsf{plonkish}}$ contains all valid instances.
-
-$$
-\mathcal{R}_{\mathsf{plonkish}} =
-\left\{ \begin{array}{cc | c}
-    (\mathbb{F}, (f,  \phi), \equiv_A, S_I, \{\mathsf{CUS}_{u}, p_u\}_u, \{ \mathsf{LOOK}_v, \mathsf{TAB}_v, q_v \}), & &  \\
-   w \in \mathbb{F}^{m_A \times n}, f \in \mathbb{F}^{m_F \times n}, \phi \in \mathbb{F}^{m_I \times n} & & \\
-   \equiv_A \subset ([0,m_A) \times [0,n)) \times ([0,m_A) \times [0,n)) & & (i,j) \equiv_A (k,\ell) \Rightarrow w[i, j] = w[k, \ell] \\
-   S_I \subset [0,m_A) \times [0,n) \times [0,m_I) \times [0,n) & & (i,j,k,\ell) \in S_I \Rightarrow w[i, j] = \phi[k, \ell] \\
-   S_F \subset [0,m_A) \times [0,n) \times [0,m_F) \times [0,n) & & (i,j,k,\ell) \in S_F \Rightarrow w[i, j] = f[k, \ell] \\
-   \forall u, \ \mathsf{CUS}_u \subset [0,n), \ p_u: \mathbb{F}^{m_F + m_A} \mapsto \mathbb{F} & &  j \in \mathsf{CUS}_u \Rightarrow p_u( f[0, j], \ldots ,  f[m_F-1, j], w[0, j], \ldots, w[m_A-1,j] ) = 0 \\
-   \mathsf{LOOK}_v \subset  [0,n),  \mathsf{TAB}_v \subset \mathbb{F}^{m_F}  & & j \in \mathsf{LOOK}_v \Rightarrow q_v(w[0, j], \ldots, w[m-1, j] ) \in \mathsf{TAB}_k \\ 
-\end{array} \right\}
-$$
-
-The relation $\mathcal{R}_{\mathsf{plonkish}}$ takes as public inputs the instances 
-$$ \mathsf{instance} = (\mathbb{F}, (f,  \phi), S_A, S_I, \{\mathsf{CUS}_{k}, p_k\}_k, \{ \mathsf{LOOK}_k, \mathsf{TAB}_k \})$$  of the following form.
-
-### Inputs into $\mathcal{R}_{\mathsf{plonkish}}$ 
+The relation $\mathcal{R}\_{\mathsf{plonkish}}$ contains pairs of public instances $\mathsf{instance}$ and private advice $w$.  We say that $\mathsf{instance}$ is a valid instance whenever their exists some advice $w$ such that $(\mathsf{instance}, w) \in \mathcal{R}\_{\mathsf{plonkish}}$.  The Plonkish language $\mathcal{L}\_{\mathsf{plonkish}}$ contains all valid instances.
```

Comment:
The `\_` approach works to avoid incorrect formatting of multiple subscripts on GitHub, but not on HackMD. Using `_ ` (underscore space) works on both.

Reply:
good point, changed